site/usage/compile.md

```diff
@@ -24,11 +24,12 @@ If provided, the `options` argument should be an object with one or more of the
 
 * [`config`](#config) sets a default config
 * [`logger`](#logging) sets a logger
+* ['fieldTitle`](#field-title) sets a field title formatter
 
 {:#config}
 ### Customized Configuration
 
-You can specify a [config]({{site.baseurl}}/docs/config.html) object, which is used as the default config. Configurations in the Vega-Lite specification itself override what is passed in through the `compile` function.
+You can specify a [config]({{site.baseurl}}/docs/config.html) object as a property of the `compile` function's `options` argument. This `config` property will be used as the default configurations. Configurations in the Vega-Lite specification will override the `config` passed in through the `compile` function.
```

Comment:
"The `config` property will be used as the default configuration.", lol

Reply:
"The configuration in the Vega-Lite specification will override the `config` passed in through the `compile` function."